apis/app/definition.yaml

```diff
@@ -3,6 +3,7 @@ kind: CompositeResourceDefinition
 metadata:
   name: xapps.azure.platformref.upbound.io
 spec:
+  defaultCompositeDeletePolicy: Foreground
```

Comment:
Do we document somewhere why we stick with `Foreground` by default?

Reply:
like this: https://github.com/upbound/platform-ref-aws/pull/113#discussion_r1312079781
otherwise usages not working when using XRs

Reply:
yep, can we make a comment?

Reply:
done